services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -66,7 +66,7 @@ async def open_project(request: web.Request) -> web.Response:
             request.app,
             project_uuid=f"{path_params.project_id}",
             user_id=req_ctx.user_id,
-            include_state=True,
+            include_state=False,
```

Comment:
are you sure about this?

Reply:
yes in the end we use the function projects_api.add_project_states_for_user() which rewrites the state, till then the state is not used.

Reply:
> I actually think we should have maybe a new column with `last_opened` in the projects table. What do you think @pcrespov, @GitHK. I think using last_modified like this is wrong. We are using a well defined entry for something different.

I think it is not used for something different, the main purpose of this issue was that when user opens and closes study without any change so the last modified timestamp will be updated. The reason for this is when he is in the dashboard view by default it is sorted by last modified timestamp, therefore his last open/closed study will be in the first place.

The only thing this PR is doing is that when the study is open, it artificially updated the last modified timestamp (which is done automatically when updating the row). An alternative to this would be to deal with this on the front end, and when the study is opened frontend automatically sends an artificial update of the study.

Reply:
As discussed in person, for now, we will go the easy way and modify the current **last change date** when the project is opened. Usually, when the user opens the study he already makes some modifications, so, in the end, it would be changed either way. If in the future somebody will complain about this behavior, we will introduce a new table with a column **last open date**, which can be set as a default sorting mechanism in the dashboard.